test/collect_coverage_api_test.dart

```diff
@@ -99,10 +115,27 @@ Future<Map<String, dynamic>> _collectCoverage(
         serviceUriCompleter.complete(serviceUri);
       }
     }
+    if (line.contains('isolateId = ')) {
+      isolateIdCompleter.complete(line.split(' = ')[1]);
+    }
   });
+
   final Uri serviceUri = await serviceUriCompleter.future;
-  final Set<String> isolateIdSet = isolateIds ? Set() : null;
+  final String isolateId = await isolateIdCompleter.future;
+  final Set<String> isolateIdSet =
+      isolateIds ? Set.from(<String>[isolateId]) : null;
```

Comment:
Prefer the `Set<E>.of(Iterable<E>)` [constructor](https://api.dartlang.org/stable/2.5.1/dart-core/Set/Set.of.html) introduced in Dart 2, which has the nice benefit of type inference, so you can just do `Set.of([isolateId])` without explicitly having to specify the generic.

Reply:
Good to know! Updated!